src/Containers/Microsoft.NET.Build.Containers/Registry/Registry.cs

```diff
@@ -167,6 +171,17 @@ public async Task<ImageBuilder> GetImageManifestAsync(string repositoryName, str
                 BaseUri,
                 unknownMediaType))
         };
+
+        async Task<ManifestV2> ReadManifest()
```

Comment:
super-nit: Should have the `Async` suffix by convention?

Reply:
IMO for local functions these kinds of conventions are less useful. You have all of the context already inside the parent function.